packages/create-flex-plugin/src/utils/validators.ts

```diff
@@ -86,16 +45,16 @@ export const _promptForTemplateUrl = async (): Promise<string> => {
 const validate = async (config: FlexPluginArguments): Promise<FlexPluginArguments>  => {
     config.name = config.name || '';
 
-    if (!_isValidPluginName(config.name)) {
+    if (!isValidPluginName(config.name)) {
         logger.error('Invalid plugin name. Names need to start with plugin-');
         return process.exit(1);
     }
 
-    if (!isSidOfType(config.accountSid, 'AC')) {
+    if (!isSidOfType(config.accountSid, SidPrefix.AccountSid)) {
```

Comment:
out of scope for this PR, but feel like SidPrefix.AccountSid should have be named like a static constant (all caps, either just keys or both SidPrefix and AccountSid)